### PR TITLE
allow php 7.2

### PR DESCRIPTION
**Pull request by cbergmann**
#### Pull Request (PR) description
Sury supports php 7.2 for debian. As php 7.1 (and 7.0) will be discontinued end of this year it might be needed to migrate to 7.2. As 7.0 is the only supported Version in official Debian 9 repos. I think that this Module should support PHP 7.2 from sury.

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

File: manifests/globals.pp [hunk 1]
```diff
@@ -77,6 +77,7 @@
         }
       } else {
         case $globals_php_version {
+          /^5\.6/,
```
Comment (igalic) [inline manifests/globals.pp]: where was this before?

Comment (igalic) [reply manifests/globals.pp]: yupp, that's what travis is showing https://travis-ci.org/voxpupuli/puppet-php/jobs/391234923#L1338

Comment (cbergmann) [reply manifests/globals.pp]: so what should We Do? Drop Debian 7 Support or more code?

File: manifests/repo/debian.pp [hunk 2]
```diff
@@ -72,17 +72,17 @@
...
-    # Add PHP 7.1 key + repository
-    apt::key { 'php::repo::debian-php71':
+    # Add PHP sury key + repository
+    apt::key { 'php::repo::debian-php-sury':
       id     => 'DF3D585DB8F0EB658690A554AC0E47584A7A714D',
       source => 'https://packages.sury.org/php/apt.gpg',
     }
 
-    ::apt::source { 'source_php_71':
+    ::apt::source { 'source_php_sury':
```
Comment (bastelfreak) [inline manifests/repo/debian.pp]: Is that the name of the created file in `/etc/apt/sources.list.d/`? So people would end up with two files that contain the same repo?

File: manifests/globals.pp (hunk 1 above)
Comment (bastelfreak) [reply manifests/globals.pp]: Everything that is Debian 7 specific can go away. We don't support it anymore. But we still have to support PHP 5.6.

File: manifests/repo/debian.pp (hunk 2 above)
Comment (dsampaolo) [reply manifests/repo/debian.pp]: I think it's a good idea to remove the "7.1" mark in the repo name. Sury is stable and will be used too for future versions.

File: manifests/globals.pp (hunk 1 above)
Comment (bastelfreak) [reply manifests/globals.pp]: actually, we could drop PHP 5.6 now.

**Pull request merged**